go/vrf/conversion.go

```diff
@@ -55,15 +55,14 @@ func i2osp(x *big.Int, rLen uint) []byte {
 // elliptic curve point type and instead directly treat the EC point as an
 // octet string per above encoding.  When using such an implementation, the
 // point_to_string function can be treated as the identity function.)
-func secg1EncodeCompressed(curve elliptic.Curve, x, y *big.Int) []byte {
+func marshalCompressed(curve elliptic.Curve, x, y *big.Int) []byte {
```

Comment:
Can we use the Golang implementation for this (https://github.com/golang/go/blob/f7ba82d68f90e20aa9e6aa973cb6f12321abec71/src/crypto/elliptic/elliptic.go#L324)?

Reply:
Yes, I very much would like to, but this function won't be available till go 1.15 which isn't released yet.

Reply:
Ah I see, maybe it's best then to just use the exact same implementation. They look correct, but it's tricky to check all the details.

Reply:
Yep that's what I'm doing in #5